test/param/ferguson/tryresolve-user.chpl

```diff
@@ -0,0 +1,92 @@
+
+
+proc hi0() {
+  writeln("hi");
+}
+
+proc hi1(x:int) {
+  writeln("hi", x);
+}
+
+proc hi2(x:int, y) {
+  writeln("hi", x);
+}
+
+
+var x:bool;
+
+x = canResolve("hi0");
+assert(x);
```

Comment:
No, it doesn't matter. That's just how I wrote it. Sometimes it's nice to have different line numbers for the new call itself.